task_1_2_2_/src/test/java/MyTreeTest.java

```diff
@@ -1,76 +0,0 @@
-import ccfit.nsu.r.masud.MyTree;
-import org.junit.jupiter.api.BeforeAll;
```

Comment:
I'm not sure that this lab should delete files from other labs

Maybe you used git rm incorrectly?